src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/SignatureHelper.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+
+namespace System.Reflection.Emit
+{
+    // TODO: Only support simple signatures. More complex signatures will be added.
+    internal static class MetadataSignatureHelper
+    {
+        internal static BlobBuilder FieldSignatureEncoder(Type fieldType)
+        {
+            var fieldSignature = new BlobBuilder();
```

Comment:
nit\style. Not sure if there is a style guideline for this, but
`BlobBuilder fieldSignature = new();`
instead of
`var fieldSignature = new BlobBuilder ();`
is more instantly readable to me.

Applies to other locations as well.